src/OpenRiaServices.Server.EntityFrameworkCore/Framework/EFCoreDescriptionProvider.cs

```diff
@@ -57,11 +65,16 @@ public override ICustomTypeDescriptor GetTypeDescriptor(Type objectType, ICustom
             return td;
         }
 
-        public override bool LookupIsEntityType(Type type)
-        {
+        public override bool LookupIsEntityType(Type type) =>
+#if NETSTANDARD2_0
             // EF6 excludes "complex objects" here so we exclude owned entities
-            return _typeDescriptionContext.GetEntityType(type) is IEntityType entityType
+            _typeDescriptionContext.GetEntityType(type) is IEntityType entityType
                 && !entityType.IsOwned();
-        }
+#else
+            // EF6 excludes "complex objects" here so we exclude owned entities
+            _typeDescriptionContext.GetEntityType(type) is IReadOnlyEntityType entityType
+                && !entityType.IsOwned();
+#endif
```

Comment:
Did you know you can use the somewhat obscure C#10 pattern `{ }` to check for null and declare a variable at the same time. It works like one would have expected "is var X" to do (but the later is unexpectedly true for null which is why it listed the interface type)

`
            _typeDescriptionContext.GetEntityType(type) is {} entityType
                && !entityType.IsOwned();
`